### PR TITLE
Config cleanup (by Johannes)

### PR DESCRIPTION
**Pull request by inaod568**
Reverting the revert.

### CODE REVIEW

File: dds_web/__init__.py [hunk 1]
```diff
@@ -94,7 +94,10 @@ def setup_logging():
 def create_app():
     """Construct the core application."""
 
-    # App config file
+    # Default development config
+    app.config.from_object("dds_web.config.Config")
```
[the remark this answers is not in the document]
Comment (alneberg) [reply dds_web/__init__.py:98]: Any deployment open for more than localhost need to override this config with environment variables or with the app config file.

[the remark this answers is not in the document]
Comment (inaod568) [reply dds_web/__init__.py:98]: @hamzaimran08

[the remark this answers is not in the document]
Comment (hamzaimran08) [reply dds_web/__init__.py:98]: I agree, this is unsafe. For the deployment we can use env variables.

[the remark this answers is not in the document]
Comment (alneberg) [reply dds_web/__init__.py:98]: Yes of course its unsafe to keep config in the github repo, but the reasoning is that we want to run a localhost server without creating any new config files. This is never supposed to be used outside of a localhost development setting.

**Comment by inaod568** [comment]
@hamzaimran08 I think I can merge this right, since it's not actually an automatic deployemnt and just publishing to dockerhub so any deployment variables will be overridden anyway?

**Comment by hamzaimran08** [comment]
> @hamzaimran08 I think I can merge this right, since it's not actually an automatic deployemnt and just publishing to dockerhub so any deployment variables will be overridden anyway?

Yes, I think then this can be merged.

**Pull request merged**